### PR TITLE
[471147] make StringConcatenationClient.appendTo() accessible

### PR DESCRIPTION
**Pull request by meysholdt**
Signed-off-by: Moritz Eysholdt <moritz.eysholdt@itemis.de>

### CODE REVIEW

**Comment by eclipsewebmaster** [comment]
Issue tracker reference:
https://bugs.eclipse.org/bugs/show_bug.cgi?id=471147

File: plugins/org.eclipse.xtext.xbase.lib/src/org/eclipse/xtend2/lib/StringConcatenationClient.java [hunk 1]
```diff
@@ -35,6 +35,20 @@
...
+	 * Delegates to {@link #appendTo(TargetStringConcatenation)} to make it publicly accessible.
+	 * 
+	 * @param client
+	 *            the {@link StringConcatenationClient} on which {@link #appendTo(TargetStringConcatenation)} is
+	 *            invoked.
+	 * @param target
+	 *            the {@link TargetStringConcatenation} to-append-to. Never <code>null</code>.
+	 * @since 2.8
+	 */
+	public static void appendToClient(StringConcatenationClient client, TargetStringConcatenation target) {
```
Comment (szarnekow) [inline plugins/org.eclipse.xtext.xbase.lib/src/org/eclipse/xtend2/lib/StringConcatenationClient.java]: The method name suggests that something is appended to the client though its the other way round.

**Comment by szarnekow** [comment]
Can we come up with a better method name?

**Comment by dhuebner** [comment]
append(client,target)
appendClientToTarget(client,target)

I prefer the first because it's  shorter

**Comment by JanKoehnlein** [comment]
Sorry, I messed it up. Did a rebase before pushing

**Comment by JanKoehnlein** [comment]
Closing this and reopening the rebased one.

**Pull request closed without merging**